vehicle/Jeep.h

```diff
@@ -5,8 +5,8 @@
 
 class Jeep final : public Car {
 public:
-    Jeep(IEngine* engine, IGear* gear)
-        : Car(engine, gear) {}
+    Jeep(IEngine* m_engine, IGear* m_gear)
```

Comment:
same here

Reply:
updated, p_ for parameters, m_ for class members